exercises/resistor-color/test/test_resistor_color.c

```diff
@@ -1,7 +1,10 @@
+#include <assert.h>
 #include "vendor/unity.h"
 #include "../src/resistor_color.h"
 
 #define ARRAY_LENGTH(A) (sizeof(A) / sizeof(A[0]))
+static_assert(sizeof(resistor_band_t) == sizeof(int),
+              "We expected the size to be the size of int");
```

Comment:
`static_assert` doesn't exist in C which is why you're failing CI. How about changing this to a check in one of the tests?

Reply:
In [newer version of libc](https://code.woboq.org/userspace/glibc/assert/assert.h.html#138) this works (I tested it, I realy did!)

Reply:
Unfortunately that doesn't help, since we need to support old versions.